### DISCUSSION TITLE
edit bim file

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1292] [Q&A] [answered]
who can tell me how edit the bim file (change from iTwin Snapshot),
for example
1.  the bim file  location（geographical coordinates）, 
2. a main revit file have some reference revit file ，when i find the reference revit file not right location in main  revit file,i must edit the revit file ,and change all these file to a bim file。

**Comment by pmconne** [comment, marked as answer]
Snapshots are not for editing. Use [iTwin Synchronizer](https://www.bentley.com/en/products/product-line/digital-twins/itwin-synchronizer) to incrementally synchronize your iModel with your source files (revit, dgn, etc).
